projects/igniteui-angular/src/lib/grids/grid-base.directive.ts

```diff
@@ -1859,8 +1859,6 @@ export abstract class IgxGridBaseDirective extends DisplayDensityBase implements
     }
 
     /**
-     * @deprecated in version 12.1.0. Use `page` property form `paginator` component instead
```

Comment:
This should be marked internal.

Reply:
We ended up keeping it deprecated as-is, right? So those two comments are now outdated, right?

Reply:
If this is so (keeping the deprecated props/methods as-is), is this valid for the outputs too? Like still emitting `perPageChange` grid event. My expectation was those will be removed.

Reply:
Those are not used internally.

Reply:
After discussing this with @ddincheva and @damyanpetev we are going to merge this PR as is, and do separate PR changes for the deprecations. (This is only API refactoring)